posthog/api/test/test_organization.py

```diff
@@ -1,17 +1,9 @@
-import datetime
-import uuid
-from typing import cast
 from unittest.mock import patch
 
-import pytest
-import pytz
-from django.core import mail
 from rest_framework import status
 
-from posthog.models import Dashboard, Organization, OrganizationMembership, Team, User
-from posthog.models.organization import OrganizationInvite
+from posthog.models import Organization, OrganizationMembership, Team, User
```

Comment:
Only change in this file was removing the signup-related tests and moving them to the new signup-specific file.